app/frontend/basic-auth.js

```diff
@@ -69,7 +57,7 @@ const setBasicAuth = (app, funcLogger) => {
       logger.info('2FA authentication is enabled. Add 2FA option.');
       basicAuthOptions.realm += ' - 2FA';
     }
-
+    console.log('basicAuthOptions => ', basicAuthOptions);
```

Comment:
Floating console.log

Reply:
Oh I think you reviewed outdated code. Please check again. :)